zun_kata.md

```diff
@@ -0,0 +1,117 @@
+# Introduction
+
+This guide describes how to get Kata containers to work with OpenStack Zun using DevStack on Ubuntu 16.04. Running DevStack with this guide will setup Docker and Clear Containers 2.0, which you replace with Kata Containers. Currently, the instructions are based on the following links:  
+https://docs.openstack.org/zun/latest/contributor/quickstart.html
+
+https://docs.openstack.org/zun/latest/admin/clear-containers.html
+
+https://github.com/kata-containers/documentation/blob/master/install/ubuntu-installation-guide.md
+
+
+# Install git to use with devstack
+
+```
+sudo apt install git
+```
+
+# Setup OpenStack DevStack
+
+```
+sudo mkdir -p /opt/stack
+sudo chown $USER /opt/stack
+git clone https://git.openstack.org/openstack-dev/devstack /opt/stack/devstack
+HOST_IP="$(ip addr | grep 'state UP' -A2 | tail -n1 | awk '{print $2}' | cut -f1 -d'/')"
+git clone https://git.openstack.org/openstack/zun /opt/stack/zun
+cat /opt/stack/zun/devstack/local.conf.sample \
+    | sed "s/HOST_IP=.*/HOST_IP=$HOST_IP/" \
+    > /opt/stack/devstack/local.conf
+sed -i "s/KURYR_CAPABILITY_SCOPE=.*/KURYR_CAPABILITY_SCOPE=local/" /opt/stack/devstack/local.conf
+echo "ENABLE_CLEAR_CONTAINER=true" >> /opt/stack/devstack/local.conf
+/opt/stack/devstack/stack.sh
+source /opt/stack/devstack/openrc admin admin
+```
+
+The previous commands start OpenStack DevStack with Zun support. You can test it using runc as shown by the following commands to make sure everything installed correctly and is working.
+
+```
+zun run --name test cirros ping -c 4 8.8.8.8
+zun list
+zun logs test
+zun delete test
+```
+
+# Install kata-runtime, kata-shim, and kata-proxy
```

Comment:
I would replace "Install kata-runtime, kata-shim, and kata-proxy" with "Install `kata-runtime`, `kata-shim`, and `kata-proxy`"